analyzers/dataframe/CMakeLists.txt

```diff
@@ -17,11 +17,29 @@ include_directories(${EDM4HEP_INCLUDE_DIRS}
                     ${VDT_INCLUDE_DIR}
                     )
 
-message(STATUS "includes-------------------------- dataframe awkward: ${AWKWARD_INCLUDE}")
-message(STATUS "includes-------------------------- dataframe edm4hep: ${EDM4HEP_INCLUDE_DIRS}")
-message(STATUS "includes-------------------------- dataframe podio  : ${podio_INCLUDE_DIR}")
-message(STATUS "includes-------------------------- dataframe fastjet: ${FASTJET_INCLUDE_DIRS}")
-
+message(STATUS "includes-------------------------- dataframe awkward:     ${AWKWARD_INCLUDE}")
+message(STATUS "includes-------------------------- dataframe edm4hep:     ${EDM4HEP_INCLUDE_DIRS}")
+message(STATUS "includes-------------------------- dataframe podio  :     ${podio_INCLUDE_DIR}")
+message(STATUS "includes-------------------------- dataframe fastjet:     ${FASTJET_INCLUDE_DIRS}")
+
+set(EXTRA_INCLUDE_DIRS)
+set(EXTRA_LIBRARIES)
+
+if(${WITH_ONNX})
+  find_package(ONNXRuntime)
```

Comment:
```suggestion
  find_package(ONNXRuntime REQUIRED)
```

Reply:
I would avoid setting it as REQUIRED here, as it might possibly set the WITH_ONNX flag back to OFF if either of the two libraries is not found.